### PR TITLE
Update truffle config and readme to support polygon to enable console GNFT PFA transfers

### PR DESCRIPTION
**Pull request by DeusExM4ch1n4**
The readme has updated instructions on how to perform a transfer of a SHARE smart contract asset from one address to another. To support this readme context the truffle config was updated to include a polygon config for mainnet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,76 @@ truffle(optimism_goerli)> share_instance.addApprovedBuild("0x0000000000000000000
 }
 ```
 
+## Editing contract ownership and configuration
```
Comment (un7c0rn) [inline README.md]: Let's move these instructions to internal README.

File: truffle-config.js [hunk 2]
```diff
@@ -27,14 +33,31 @@ module.exports = {
...
+        ),
       network_id: 80001,
       gas: 5500000,
       confirmations: 2,
       timeoutBlocks: 200,
       networkCheckTimeout: 1000000,
       skipDryRun: true,
     },
+
+    polygon: {
```
Comment (un7c0rn) [inline truffle-config.js:48]: This LGTM.

**Review by un7c0rn — CHANGES_REQUESTED** [review]
Thanks JP! Notes inline.

**Comment by DeusExM4ch1n4** [comment]
@un7c0rn The readme updates are already in the internal readme, would you like me to remove them here?

**Comment by un7c0rn** [comment]
> @un7c0rn The readme updates are already in the internal readme, would you like me to remove them here?

@DeusExM4ch1n4 Ok SG. Yea, these are good instructions to have as internal business processes but out of scope for the OSS repo.

**Comment by DeusExM4ch1n4** [comment]
> > @un7c0rn The readme updates are already in the internal readme, would you like me to remove them here?
> 
> @DeusExM4ch1n4 Ok SG. Yea, these are good instructions to have as internal business processes but out of scope for the OSS repo.

@un7c0rn  should the logic about using truffle console also be removed?

**Review by un7c0rn — APPROVED** [review]
LGTM

**Pull request merged**